src/styles/style.css

```diff
@@ -0,0 +1,92 @@
+@import url("https://peaceui.netlify.app/peace.css");
+   
+:root{
+    --primary: #1F1D2B;
+    --primary-100: #2F3142;
+
+    --white: #fff;
+}
+
+body{
+    color: var(--white);
+    background-color: var(--primary);
+}
+
+.nav-wrapper{
+    background-color: var(--primary);
+    width: 100%;
+}
+
+.btn{
+    background-color: var(--primary-100);
+    margin: 0;
+}
+
+.category-title{
+    margin: 0.5em 0em 0.7em 0em;
+}
+
+.category{
+    margin: 2em
+}
+
+.category-grid{
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 2em;
+}
+
+.card{
+    background-color: var(--primary-100);
+    color: var(--white);
+    cursor: pointer;
+}
+
+.card-img{
+    position: relative;
+}
+
+.hover-icon{
+    position: absolute;
+    font-size: 3em;
+    left:50%;
+    top:50%;
+    transform: translate(-50%, -50%);
+}
+
+.card-head>.card-title{
+    font-size: 1.3rem;
+}
+
+.card-subtitle{
+    opacity: 50%;
+}
+
+.card-option{
+    height: 0;
+}
+
+.card-content{
+    display: flex;
+    justify-content: baseline;
+    align-items: baseline;
+}
+
+.card-actions{
+    margin-left: auto;
+    margin-right: 0.5em;
```

Comment:
you can use pixel instead of em

Reply:
trying to keep it relative to fontsize. Will surely switch if UI breaks at some point.